compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.td

```diff
@@ -289,6 +289,8 @@ def IREEGPU_TargetWgpAttr : AttrDef<IREEGPU_Dialect, "TargetWgp"> {
     "uint32_t":$max_thread_count_per_workgroup,
     // The maximal number of shared memory bytes we can allocate per workgroup.
     "uint32_t":$max_workgroup_memory_bytes,
+    // Tthe maximum number of workgroups per X/Y/Z dimension in a dispatch.
```

Comment:
The `TargetWGPAttr` is meant for features and limits in a single GPU workgroup processor. The max workgroup counts actually is more suitable at the `TargetChipAttr` where features and limits are for the whole chip?

Reply:
Except that `TargetChipAttr` is for things that vary within a chipset, like the number of compute units. And the number of workgroups available sure is an attribute of a workgroup processor.

Reply:
@antiagainst Pinging again on this

Reply:
Fair enough.